src/io/aleph/dirigiste/Pool.java

```diff
@@ -373,7 +382,7 @@ private void startControlLoop(int duration, int iterations) {
                 Thread.sleep(Math.max(0, duration - (System.currentTimeMillis() - start)));
             }
         } catch (InterruptedException e) {
-
+            // FIXME: What happens if it crashes?...
         }
```

Comment:
Probably need a `finally` clause that checks `isHeldByCurrentThread` and calls `unlock` if true.

Reply:
01092d533476facdfa14073832cd24cc13b6527d

Reply:
No longer relevant.